src/mixins/FetchFacesMixin.js

```diff
@@ -76,8 +75,8 @@ export default {
 				this.loadingFaces = true
 				this.errorFetchingFaces = null
 
-				const { data: faces } = await client.getDirectoryContents(`/recognize/${getCurrentUser()?.uid}/faces/`, {
-					data: DavRequest,
+				const { data: faces } = await davClient.getDirectoryContents(`/recognize/${getCurrentUser()?.uid}/faces/`, {
```

Comment:
Would it make sense to use `davGetClient()` everywhere ?

```suggestion
				const { data: faces } = await davGetClient().getDirectoryContents(`/recognize/${getCurrentUser()?.uid}/faces/`, {
```